### DISCUSSION TITLE
Recursion?

### DISCUSSION THREAD

**Discussion by brurucy** [EvgSkv/logica#74] [Ideas]
It seems that Logica does not support recursion, is that so?

**Comment by EvgSkv** [comment]
That's right, recursion is not supported _yet_.
However an example of solving a problem that is traditionally solved via recursion can be found [here](https://github.com/EvgSkv/logica/blob/main/examples/scripts/closure_use.l).
We plan to support basic recursion soon.

**Comment by EvgSkv** [comment]
@brurucy Support for recursion has been added! (Some time ago actually)
E.g. see:
https://twitter.com/EvgSkvDev/status/1396168603217448962
